src/containers/Nav/Nav.js

```diff
@@ -4,10 +4,14 @@ import './Nav.css'
 
 const Nav = () => {
     return(
-        <nav>
-            Testing Nav
+        <nav className="nav">
+            <p className="nav-p">FAVORITES</p>
+            <div className="nav-user-options">
+              <p className="nav-p">LOGIN</p>
```

Comment:
I believe these are also p tags for styling, but semantically I feel like they should be buttons, at the very least for accessibility.

Reply:
If we do not change these, then at least add aria roles.